projects/packages/forms/src/contact-form/js/accessible-form.js

```diff
@@ -78,11 +78,14 @@ const initForm = form => {
 		if ( isFormValid( form ) ) {
 			inputListenerMap = {};
 
+			form.removeEventListener( 'submit', onSubmit );
```

Comment:
This wasn't required in the front end since submitting a valid form triggered a hard page refresh. However, we need it here to prevent an infinite loop.